client/src/components/ReviewForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorList"
+
+const ReviewForm = ({ postReview, errors, shoe, user, setErrors }) => {
+  const [newReview, setNewReview] = useState({
+    body: "",
+    score: "",
+    userId: user.id
```

Comment:
the user is managed through the passport session in express, so we don't need to also send it in the request with the form. the front end forms should be responsible for just getting data from the user. once the form has been submitted and received on the backend we can attach the currently signed in user - see this article!